content/blog/angular-extend-class/index.md

```diff
@@ -959,11 +959,11 @@ class WindowSizeService {
         height: window.innerHeight,
         width: window.innerWidth,
       })),
-      takeUntil(this.destroy$),
       startWith({
         height: window.innerHeight,
         width: window.innerWidth,
       })
```

Comment:
Missing a comma now, lets update and remove the other comma

Reply:
Good catch!